clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -811,6 +826,16 @@ private void handleFilesetCommand() {
     }
   }
 
+  private boolean hasEntity(FullName name) {
+    // TODO fileset and topic
+    return !(Objects.isNull(name.getCatalogName())
+        && Objects.isNull(name.getSchemaName())
+        && Objects.isNull(name.getTableName()));
+  }
+
+  private void showEntityUndefinedMessage() {
+    System.err.println(ErrorMessages.MISSING_ENTITY);
+  }
```

Comment:
I would just use System.err.println(ErrorMessages.MISSING_ENTITY) directly where needed as it clearer to what it is doing.

Reply:
Since the warning is identical in all three places, I want to centralize it to make future updates to the message easier. should we keep this method?

Reply:
I would not keep it as it doesn't add any value, if it did more than just print out that message, then sure.